blog/2021/08/juliacon-highlights.md

```diff
@@ -0,0 +1,53 @@
+@def title =  "JuliaCon 2021 Highlights"
+@def authors = "The JuliaCon 2021 committee"
+@def published = "20 August 2021"
+@def rss_pubdate = Date(2021, 8, 20)
+@def rss = """JuliaCon 2021 Highlights"""
+
+This year’s [JuliaCon](https://www.youtube.com/playlist?list=PLP8iPy9hna6Q343_8sSq4f306VGLW4TLK) was the biggest and best ever, with [more than 300 presentations available for free on YouTube](https://www.youtube.com/playlist?list=PLP8iPy9hna6Q343_8sSq4f306VGLW4TLK), more than 20,000 registrations, and more than 43,000 unique YouTube viewers during the conference.
```

Comment:
Can we have a comparison with numbers from last year, to show an increase (there has been an increase, right? :sweat_smile:)

Reply:
https://julialang.org/blog/2020/08/juliacon-2020-wrapup/